README.md

```diff
@@ -1,65 +1,75 @@
-
-## How to set up and build Bouncer tests locally
-
-This repository contains tests suite used to test Mozilla's Bouncer. Mozilla maintains a guide to run automated tests on our [QMO website](https://quality.mozilla.org/docs/webqa/running-webqa-automated-tests/).
-
-You will need to install the following:
-
-* **Git**: If you have cloned this project already then you can skip this! GitHub has excellent guides for [Windows](https://help.github.com/articles/set-up-git/#platform-windows), [OS X](https://help.github.com/articles/set-up-git/#platform-mac) and [Linux](https://help.github.com/articles/set-up-git/#platform-linux).
-* **Python**: Before you will be able to run these tests you will need to have [Python 2.6](https://www.python.org/download/releases/2.6/) installed.
-
-### Installing `pip` (for managing Python packages)
-
-```bash
-sudo easy_install pip
-```
-
-### Installing dependencies
-
-If you are using `virtualenv`, run the following in the project root:
-
-```bash
-virtualenv venv
-source venv/bin/activate
-pip install -r requirements.txt
-```
-
-If you are not using `virtualenv`, run the following in the project root to install dependencies globally:
+While we take pains to keep our documentation updated, the best source of
+information is those of us who work on the project. Don't be afraid to join us
+in irc.mozilla.org [#mozwebqa][irc] to ask questions about our tests. We also
+have a [mailing list][list] available that you are welcome to join and post to.
+
+## How to run the tests locally
+We maintain a [detailed guide][running-tests] to running our automated tests.
+However, if you want to get started quickly, you can try following the steps
+below:
+
+### Clone the repository
+If you have cloned this project already then you can skip this, otherwise you'll
+need to clone this repo using Git. If you do not know how to clone a GitHub
+repository, check out this [help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining
+them in the future, it would be a good idea to create a fork of this repository
+first, and then clone that. GitHub also has great documentation for
+[forking a repository][git-fork].
+
+### Create or activate a Python virtual environment
+You should install this project's dependencies (which is described in the next
+step) into a virtual environment in order to avoid impacting the rest of your
+system, and to make problem solving easier. If you already have a virtual
+environment for these tests, then you should activate it, otherwise you should
```

Comment:
Uber-nit (and this might exist in the original README we devised and Bob put up): s/activate it, otherwise/activate it; otherwise/ (just to keep those two clauses separate a bit).  Feel free to leave as-is, though: just a personal-style nit.